src/Core/Content/ProductExport/ScheduledTask/ProductExportGenerateTaskHandler.php

```diff
@@ -45,13 +52,13 @@ public function run(): void
         foreach ($salesChannelIds as $salesChannelId) {
             $productExports = $this->fetchProductExports($salesChannelId);
 
-            if (\count($productExports) === 0) {
+            if ($productExports->count() === 0) {
                 continue;
             }
 
             $now = new \DateTimeImmutable('now');
 
-            foreach ($productExports as $productExport) {
+            foreach ($productExports->getElements() as $productExport) {
```

Comment:
```suggestion
            foreach ($productExports as $productExport) {
```
should not be necessary